vignettes/GenomeInfoDb.Rmd

```diff
@@ -0,0 +1,322 @@
+---
+title: "An Introduction to [*GenomeInfoDb*](http://bioconductor.org/packages/GenomeInfoDb)"
+date: "Modified 26 June, 2020.  Compiled  `r format(Sys.time(), '%B %d, %Y')`"
```

Comment:
The `author` line was left out.

Reply:
It looks like it is still missing. @villafup make sure you commit to the convert-genomeinfodb branch and push to https://github.com/villafup/GenomeInfoDb/tree/convert-genomeinfodb